### ISSUE TITLE
Improve colour contrast ratio

### ISSUE THREAD

**Issue by chinwobble** [praeclarum/FuGetGallery#93]
Many of the menus fail on the fuget gallery fail Web Content Accessibilty guidelines on colour contrast. 

you can check the colours here.
https://webaim.org/resources/contrastchecker/

For instance most buttons in this menu have grey buttons which is too similar to the black blackground.
![image](https://user-images.githubusercontent.com/18226001/70192000-91400000-174e-11ea-98f9-999debe9efd3.png)

![image](https://user-images.githubusercontent.com/18226001/70192151-fac00e80-174e-11ea-9c8e-11031b1923ee.png)

**Comment by praeclarum** [comment]
Cool! I will run that tool the next time I'm writing CSS.
